app/src/main/java/org/permanent/permanent/ui/compose/DesignateContactOrStewardScreen.kt

```diff
@@ -0,0 +1,172 @@
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat
+import org.permanent.permanent.R
+
+@Composable
+fun DesignateContactOrStewardScreen() {
+    val context = LocalContext.current
+    val scrollState = rememberScrollState()
+
+    val primaryColor = Color(ContextCompat.getColor(context, R.color.colorPrimary))
+    val blackColor = Color(ContextCompat.getColor(context, R.color.black))
+    val lightBlueColor = Color(ContextCompat.getColor(context, R.color.superLightBlue))
+    val whiteColor = Color(ContextCompat.getColor(context, R.color.white))
+    val regularFont = FontFamily(Font(R.font.open_sans_regular_ttf))
+    val boldFont = FontFamily(Font(R.font.open_sans_bold_ttf))
+    val semiBoldFont = FontFamily(Font(R.font.open_sans_semibold_ttf))
+    val smallTextSize = 11.sp
+    val subTitleTextSize = 16.sp
+    val titleTextSize = 19.sp
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(lightBlueColor)
+            .padding(horizontal = 32.dp)
+            .verticalScroll(scrollState),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.height(40.dp))
+        Text(
+            text = stringResource(R.string.designate_a_legacy_contact).uppercase(),
+            fontSize = smallTextSize,
+            color = primaryColor,
+            fontFamily = boldFont,
+            modifier = Modifier.align(Alignment.Start)
+        )
+        Spacer(modifier = Modifier.height(10.dp))
+        Text(
+            text = stringResource(R.string.designate_contact_title),
+            fontSize = titleTextSize,
+            color = primaryColor,
+            fontFamily = semiBoldFont,
+            modifier = Modifier.align(Alignment.Start)
+        )
+        Spacer(modifier = Modifier.height(40.dp))
+        LegacyContactCard(whiteColor, primaryColor, semiBoldFont, blackColor, regularFont, boldFont)
+        Spacer(modifier = Modifier.height(300.dp))
+        Button(
+            modifier = Modifier.fillMaxWidth().height(48.dp),
+            colors = ButtonDefaults.buttonColors(containerColor = primaryColor),
+            shape = RoundedCornerShape(8.dp),
+            onClick = {
+            }
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = stringResource(R.string.button_go_to_legacy_planning),
+                    fontSize = subTitleTextSize,
+                    fontFamily = regularFont,
+                )
+                Spacer(modifier = Modifier.width(116.dp))
```

Comment:
Instead of adding a certain width, maybe use something like:
Spacer(modifier = Modifier.weight(1.0f))
This will push the image to the right side of the view without hardcoding a specific width.